docs/gettsim_objects/input_variables.rst

```diff
@@ -101,3 +101,29 @@ household.
 +-------------------------+---------------------------------------------+--------------+
 | _`priv_rente_m`         | Amount of monthly private pension           | float        |
 +-------------------------+---------------------------------------------+--------------+
+| _`weiblich`             | If gender is female                         | bool         |
```

Comment:
```suggestion
| _`weiblich`             | True if female                              | bool         |
```

*(sorry, can't help being a bit pedantic at times)* :wink:

Reply:
changed this